pkg/kv/kvserver/helpers_test.go

```diff
@@ -551,14 +552,11 @@ func WatchForDisappearingReplicas(t testing.TB, store *Store) {
 	}
 }
 
-// ChecksumRange returns a checksum over the KV data of the given range.
-func ChecksumRange(
+// CalcReplicaDigest computes a digest of the replicated data for the given
+// range. For testing only.
+func CalcReplicaDigest(
```

Comment:
nit: mild preference for getting rid of this and exporting `calcReplicaDigest`. Test callers can just set up a `noopLimiter` in the global scope and reuse that.

Reply:
Agree. I wonder if it's a commonly used pattern, and it would make sense to have some `Unlim` limiter right in the main package.

Reply:
Doesn't appear so. This type is not widely used.

Reply:
Done. The `quotapool` limiter already has code to handle infinite `rate`, so I think it makes sense to add a package constant, similarly to how it's done in Go's limiter: https://pkg.go.dev/golang.org/x/time/rate#pkg-constants. Will split this PR in 2 commits.